src/lib/sidenav/sidenav.md

```diff
@@ -18,6 +18,14 @@ A sidenav container may contain one or two `<md-sidenav>` elements. When there a
 `<md-sidenav>` elements, each must be placed on a different side of the container.
 See the section on positioning below.
 
+
+### Sidenav vs. Drawer
```

Comment:
I wouldn't make this a separate section, just part of the section above. It also needs to be worded to introduce the drawer, otherwise us: "The difference between sidenav and drawer..." user: "wait, what's drawer?"